package/xochitl/package

```diff
@@ -8,6 +8,7 @@ url=https://remarkable.com
 pkgver=0.0.0-1
 timestamp=2020-09-24T18:48Z
 section=readers
+devices=(any)
```

Comment:
```suggestion
devices=(rm1 rm2)
```
Since we don't know what the next device we support will look like, maybe it wont have xochitl.

Reply:
I personally don't see a reason why this should ever fail on a future device. But I can see the reasoning and would be willing to apply this if you still want it.

Reply:
Maybe we should settle on the definition of `any` in the docs first: d949be4c

Reply:
I agree with the definition of `any` in that commit. Which, in that spirit, this doesn't fall under as xochitl is specific to the rm1/2 and might not exist on other devices.

Reply:
rM seems committed to keeping old devices updated while moving forward with new hardware iterations. Therefore we could expect them to only introduce software differences between devices for hardware-related motives (as we’ve seen with rM2). It looks to me that this is the assumption that divides your two views on this @LinusCDE & @Eeems. If we accept this assumption, then there is no reason why `xochitl` would suddenly become unavailable for a specific hardware revision and stay available for others. If we reject it, there could be a future where a new hardware revision gets its own interface with a different naming.

In any case, I don’t think this is such a crucial issue, since for each new hardware revision there will necessarily be some period of time where we get to experiment with the new device before we move support for it to stable. In this period of time, we’ll have the opportunity to change the `devices` parameter of the `xochitl` package if need be.

Reply:
I'm also thinking that we may decide to support more than just the reMarkable line of devices in the future. Fair enough though about being able to change the devices line in the future.